src/chanakya/services/generate_cert.py

```
@@ -44,16 +40,10 @@ def generate_self_signed_cert(
     # Save the certificate and key to files
     try:
         with open(cert_path, "wt") as cert_file:
-            cert_file.write(
-                crypto.dump_certificate(crypto.FILETYPE_PEM, cert).decode("utf-8")
-            )
+            cert_file.write(crypto.dump_certificate(crypto.FILETYPE_PEM, cert).decode("utf-8"))
         with open(key_path, "wt") as key_file:
```

Ruff (0.15.6)

[warning] Unnecessary modes, use `w`

Replace with `w`

(UP015)